Dockerfile

```
@@ -1,11 +1,17 @@
-FROM ubuntu:latest AS builder
+# check=skip=FromPlatformFlagConstDisallowed
+# Prod and CI are x86_64; pin amd64 so local (e.g. Apple Silicon) builds produce
+# a deployable artifact instead of an arm64 image the droplet can't run.
+FROM --platform=linux/amd64 ubuntu:latest AS builder
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)